### ISSUE TITLE
README: ENV delta in the examples. Which one is the right one?

### ISSUE THREAD

**Issue by pascalandy** [adferrand/docker-letsencrypt-dns#70]
It's confusing to follow the cloudflare example. In this section:

### https://github.com/adferrand/docker-letsencrypt-dns#configuring-dns-provider-and-authentication-to-dns-api
```
LEXICON_CLOUDFLARE_AUTH_USERNAME=my_cloudflare_email
LEXICON_CLOUDFLARE_AUTH_TOKEN=my_cloudflare_global_api_key
```

But then we can see:

### https://github.com/adferrand/docker-letsencrypt-dns#run-the-container
```
LEXICON_CLOUDFLARE_USERNAME=my_cloudflare_email
LEXICON_CLOUDFLARE_TOKEN=my_cloudflare_global_api_key 
```

My guess is that I should use this:

```
docker run \
  --name letsencrypt-dns \
  --volume /etc/letsencrypt/domains.conf:/etc/letsencrypt/domains.conf \
  --volume /etc/letsencrypt:/etc/letsencrypt
  --env 'LETSENCRYPT_USER_MAIL=user@mymail.com' \
  --env 'LEXICON_CLOUDFLARE_AUTH_USERNAME=user@mymail.com' \
  --env 'LEXICON_CLOUDFLARE_AUTH_TOKEN=101010101010101010101010101010' \
  adferrand/letsencrypt-dns
```

Either ways, my guess is that there is confusion in the README. 
Thanks!

**Comment by adferrand** [comment]
Hello @pascalandy!

In fact, both are valid. Several months ago, only flags of type `--auth-[keyword]` from Lexicon CLI could be set using environment variables in the form of `LEXICON_[PROVIDER]_[KEYWORD]`. I changed that to allow every CLI flag to be set, using an environment variable name taking all words from the associated CLI flag. So typically `--auth-[keyword]` is set using `LEXICON_[PROVIDER]_AUTH_[KEYWORD]`. However I kept the old pattern for retro-compatibility purposes.

Anyway, the old pattern is deprecated, and should not appear in any documentation, and having inconsistencies in `README.md` is not desirable anyway. I fix that right away.

Thanks for the report!
